nixos/modules/services/monitoring/librenms.nix

```
@@ -376,6 +391,9 @@ in
       # enable fast ping by default
       "ping_rrd_step" = 60;
 
+      # set default memory limit to 1G
+      "php_memory_limit" = lib.mkDefault 1024;
```

```suggestion
      "php_memory_limit" = lib.mkDefault "1024M";
```
otherwise people cannot use shorthands like 2G

---

This is LibreNMS config and LibreNMS expects an integer here. So shorthands will not work.

It is hardcoded to megabytes as you can see here: https://github.com/librenms/librenms/blob/47a9e2b7da2d48f9e55692894a349c0192c9b51a/includes/init.php#L109